IMPLEMENTATION_PLAN_acp_goose_backend_live_e2e.md

```
@@ -0,0 +1,29 @@
+## Stage 1: Reproduce Concrete Failures
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)